includes/widgets/rating.php

```diff
@@ -94,6 +97,34 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_responsive_control( 'alignment', [
+			'label' => esc_html__( 'Alignment', 'elementor' ),
+			'type' => Controls_Manager::CHOOSE,
+			'options' => [
+				'start' => [
+					'title' => esc_html__( 'Start', 'elementor' ),
+					'icon' => "eicon-align-$start_logical-h",
+				],
+				'center' => [
+					'title' => esc_html__( 'Center', 'elementor' ),
+					'icon' => 'eicon-align-center-h',
+				],
+				'end' => [
+					'title' => esc_html__( 'End', 'elementor' ),
+					'icon' => "eicon-align-$end_logical-h",
+				],
+			],
+			'selectors_dictionary' => [
+				'start' => '--e-rating-justify-content: flex-start;',
+				'center' => '--e-rating-justify-content: center;',
+				'end' => '--e-rating-justify-content: flex-end;',
+			],
+			'selectors' => [
+				'{{WRAPPER}} .e-rating' => '{{VALUE}}',
```

Comment:
I think we can leave out the '.e-rating' class from the selector.

```suggestion
				'{{WRAPPER}}' => '{{VALUE}}',
```

Reply:
We need to fix my initial styling inside the rating.scss.

Afterwards we can remove the '.e-rating' selector from here.

Reply:
Done